### PR TITLE
[localization] Translate global strings in spanish

### PR DESCRIPTION
**Pull request by Elegidoadedo**
This PR will add the spanish translations for keys in global section.

Issue related -> https://github.com/ToolJet/ToolJet/issues/4170

### CODE REVIEW

File: frontend/assets/translations/es.json [hunk 1]
```diff
@@ -0,0 +1,917 @@
...
+        "authorize": "Autorizar",
+        "connect": "Conectar",
+        "reconnect": "Reconectar",
+        "components": "componentes",
+        "send": "Enviar",
+        "noConnection": "no se puede conectar",
+        "connectionVerifeid": "conexión verificada",
+        "left": "Izquierda",
+        "center": "Centro",
+        "right": "Derechar",
```
Comment (LPRegen) [inline frontend/assets/translations/es.json:31]: Hi, here is a small mistake, it should be `Derecha` without the last 'r'

File: frontend/assets/translations/es.json [hunk 2]
```diff
@@ -0,0 +1,917 @@
...
+        "justified": "Justificado",
+        "host": "Huesped",
```
Comment (LPRegen) [inline frontend/assets/translations/es.json:33]: Here, it should have an accent mark on the first `e`, `huésped`. But I don't know if it is worthy to translate 'host' to Spanish, I don't know nowbody that refers to the host as 'huésped' in Spanish.

File: frontend/assets/translations/es.json (hunk 1 above)
Comment (Elegidoadedo) [reply frontend/assets/translations/es.json:31]: hell, yes. A big typo. Thanks!

File: frontend/assets/translations/es.json (hunk 2 above)
Comment (Elegidoadedo) [reply frontend/assets/translations/es.json:33]: Totally agree on this point, usually i use the english term always, never seen the literal spanish translation for host / hosting stuff, will revert to english this one.
Thanks!

File: frontend/assets/translations/languages.json [hunk 3]
```diff
@@ -2,7 +2,8 @@
   "languageList": [
   { "lang": "English", "code": "en", "nativeLang": "English" },
   { "lang": "French", "code": "fr", "nativeLang": "Français" },
+  { "lang": "Spanish", "code": "es", "nativeLang": "Español" },
   { "lang": "Italian", "code": "it", "nativeLang": "Italiano" },
   { "lang": "Indonesian", "code": "id", "nativeLang": "Bahasa Indonesia" }
   ]
-}
+}
```
Comment (withshubh) [inline frontend/assets/translations/languages.json:9]:
```suggestion
```

**Review by withshubh — CHANGES_REQUESTED** [review]
remove this extra bracket

File: frontend/assets/translations/es.json [hunk 4]
```diff
@@ -0,0 +1,917 @@
...
+        "left": "Izquierda",
+        "center": "Centro",
+        "right": "Derecha",
+        "justified": "Justificado",
+        "host": "Host",
+        "operation": "Operación",
+        "header": "HEADER",
+        "path": "PATH",
+        "query": "QUERY",
+        "requestBody": "REQUEST BODY"
```
Comment (withshubh) [inline frontend/assets/translations/es.json:38]: LGTM

**Review by withshubh — APPROVED** [review]

**Review by withshubh — APPROVED** [review]

**Pull request merged**